var/spack/repos/builtin/packages/py-pyamg/package.py

```diff
@@ -25,5 +26,7 @@ class PyPyamg(PythonPackage):
     depends_on("py-numpy@1.7:", type=("build", "run"))
     depends_on("py-scipy@0.12:", type=("build", "run"))
     depends_on("py-setuptools@42:", type="build", when="@4.2.0:")
-    depends_on("py-setuptools-scm@5:+toml", type="build", when="@4.2.0:")
```

Comment:
Is 7.1.0 also needed for 4.2.3 or only for 5.0.0?

Reply:
`spack install py-pyamg@4.2^py-setuptools-scm@7.0.5` succeeds. So does `spack install py-pyamg@4.2^py-setuptools-scm@7.0.5~toml`. So apparently, the `+toml` variant isn't required, and 7.1.0 is only required for PyAMG 5.0.0.